doc/book/hybrid.md

```diff
@@ -0,0 +1,119 @@
+# Encrypt and decrypt using hybrid cryptosystem
+
+Hybrid is an encryption mode that uses symmetric and public keys ciphers together.
+The idea is to take the advantages of the public key cryptography for sharing the
+keys and the speed of symmmetric encryption to encrypt the message.
+
+The hybrid mode is able to encrypt message for one or more receivers and can be
+used in multi user scenario, where you can limit the decryption only for some users.
```

Comment:
Rephrase to:

    Hybrid mode allows you to encrypt a message for one or more receivers, and can be used in multi-user scenarios where you wish to limit decryption to specific users.